website/server/middlewares/blocker.js

```diff
@@ -0,0 +1,69 @@
+import nconf from 'nconf';
+import {
+  Forbidden,
+} from '../libs/errors';
+import { apiError } from '../libs/apiError';
+import { model as Blocker } from '../models/blocker';
+
+// Middleware to block unwanted IP addresses
+
+// NOTE: it's meant to be used behind a proxy (for example a load balancer)
+// that uses the 'x-forwarded-for' header to forward the original IP addresses.
+
+// A list of comma separated IPs to block
+// It works fine as long as the list is short,
+// if the list becomes too long for an env variable we'll switch to Redis.
```

Comment:
This also seems odd to say, given we're supporting creation of IP blocks as MongoDB records now..? Is there still a plan to use Redis instead?